### ISSUE TITLE
Translated signup/ page mess up with provider location

### ISSUE THREAD

**Issue by moedech** [nextcloud/nextcloud.com#1101]
When using https://nextcloud.com/signup/, the closest provider is shown. As soon as I switch to https://nextcloud.com/de/signup/ or https://nextcloud.com/fr_FR/signup/, it completely mess up the radian-search. Looks like it just uses a fallback, based on the distance I would say it's the two capitals Berlin and Paris.

A specially for swiss providers this is quite bad, as we do of course provide german support, but are not shown for german visitors.

**Comment by mritzmann** [comment]
I'm not sure, but maybe @skjnldsv can tell us more about this? I also think that the signup page should only sort by geo location.

**Comment by skjnldsv** [comment]
Hey!
We're aware of the issue, but for some reason we cannot find the source!
We did a bit of digging a while ago, but after losing many hours, I gave up.
We get the ip here:
https://github.com/nextcloud/nextcloud.com/blob/3c94abc0b80e36c6e7bd7e03ecb196b4103e9a5b/lib/signup.php#L21-L39
And we match the location to a offline db here:
https://github.com/nextcloud/nextcloud.com/blob/3c94abc0b80e36c6e7bd7e03ecb196b4103e9a5b/lib/signup.php#L207

My guess is it was some kind of cache, but we tried disabling it and it did not worked :/
